core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java

```diff
@@ -37,24 +38,27 @@
 
   private static final Logger LOG = LogManager.getLogger();
   private final SignerForIdentifier<?> signerForIdentifier;
-  private final HttpApiMetrics metrics;
+  private final HttpApiMetrics httpMetrics;
+  private final SlashingProtectionMetrics slashingMetrics;
   private final Optional<SlashingProtection> slashingProtection;
   private final ObjectMapper objectMapper;
 
   public Eth2SignForIdentifierHandler(
       final SignerForIdentifier<?> signerForIdentifier,
-      final HttpApiMetrics metrics,
+      final HttpApiMetrics httpMetrics,
+      final SlashingProtectionMetrics slashingMetrics,
       final Optional<SlashingProtection> slashingProtection,
       final ObjectMapper objectMapper) {
     this.signerForIdentifier = signerForIdentifier;
-    this.metrics = metrics;
+    this.httpMetrics = httpMetrics;
+    this.slashingMetrics = slashingMetrics;
     this.slashingProtection = slashingProtection;
     this.objectMapper = objectMapper;
   }
 
   @Override
   public void handle(final RoutingContext routingContext) {
-    try (final TimingContext ignored = metrics.getSigningTimer().startTimer()) {
+    try (final TimingContext ignored = httpMetrics.getSigningTimer().startTimer()) {
```

Comment:
What about the change for http metrics to be labelled by type? Perhaps this is another PR?

Reply:
Kinda had to do it here - its small, and I didn't want to have a thing in this class called "metrics" then another called "slashingMetrics" - figured they should both be explicit in their purpose.